NeoML/src/Dnn/Layers/LoraFullyConnectedLayer.cpp

```diff
@@ -52,8 +52,9 @@ CLoraFullyConnectedLayer::~CLoraFullyConnectedLayer()
 void CLoraFullyConnectedLayer::initDropoutDesc()
 {
 	if( desc == nullptr ) {
-		desc = MathEngine().InitDropout( lora.Dropout, /*isSpatial*/false, /*isBatchwise*/false,
-			inputBlobs[0]->GetDesc(), inputBlobs[0]->GetDesc(), GetDnn()->Random().Next() );
+		desc = MathEngine().InitDropout();
```

Comment:
It would be more optimal to make here the same trick as in `CDropoutLayer`: move `MathEngine().InitDropout()` to the `Reshape()` and leave here only `MathEngine().UpdateDropout()`